artifactory_test.go

```diff
@@ -4817,3 +4817,21 @@ func assertPermissionTargetDeleted(t *testing.T, manager artifactory.Artifactory
 func cleanPermissionTarget() {
 	_ = artifactoryCli.Exec("ptdel", tests.RtPermissionTargetName)
 }
+
+func TestArtifactoryCurl(t *testing.T) {
+	initArtifactoryTest(t)
+	_, err := createServerConfigAndReturnPassphrase()
+	defer deleteServerConfig()
```

Comment:
Check no errors after configuration:
```suggestion
	defer deleteServerConfig()
	assert.NoError(t, err)
```